_data/versions.yml

```diff
@@ -1,4 +1,74 @@
 v3.0:
+- title: v3.0.3
+  note: |
+    23 February 2018
```

Comment:
26?

Reply:
Nope, the release went out on the 23rd: https://github.com/projectcalico/calico/releases/tag/v3.0.3